src/runtime/plugin.js

```diff
@@ -0,0 +1,84 @@
+import Vue from 'vue'
+import { Strapi } from '@nuxtjs/strapi/runtime'
+
+const options = <%= JSON.stringify(options) %>
```

Comment:
Quick question: how do I make my IDE accept this syntax? WebStorm doesn't seem to like it since it's exclusive to `.ejs` files.

Reply:
At the moment it is not really possible, but we are working on a lintable templating engine.